internal/token/token_test.go

```
@@ -0,0 +1,365 @@
+}
+
+// serveJWKS starts an HTTP test server serving the sealer's JWKS.
+func serveJWKS(sealer *Sealer) *httptest.Server {
+	return httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		set, err := sealer.JWKSet()
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+		w.Header().Set("Content-Type", "application/json")
+		json.NewEncoder(w).Encode(set)
```

golangci-lint (2.12.2)

[error] Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)